modules/travel_pay/app/services/travel_pay/claims_client.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+require 'securerandom'
+require_relative './token_service'
+
+module TravelPay
+  class ClaimsClient
+    ##
+    # HTTP GET call to the BTSSS 'claims' endpoint
+    # API responds with travel pay claims including status
+    #
+    # @return [TravelPay::Claim]
+    #
+    def get_claims(veis_token, btsss_token)
+      request_claims(veis_token, btsss_token)
+    end
+
+    private
+
+    def claim_headers
+      if Settings.vsp_environment == 'production'
+        {
+          'Content-Type' => 'application/json',
+          'Ocp-Apim-Subscription-Key-E' => Settings.travel_pay.subscription_key_e,
+          'Ocp-Apim-Subscription-Key-S' => Settings.travel_pay.subscription_key_s
+        }
+      else
+        {
+          'Content-Type' => 'application/json',
+          'Ocp-Apim-Subscription-Key' => Settings.travel_pay.subscription_key
+        }
+      end
+    end
```

Comment:
How do you feel about creating a base `TravelPay::Client` class that includes the `connection` and `claim_headers` (need a new name for this)?

Then, we could inherit from the base client class and save ourselves some repeated code and future headache.

Reply:
I like that idea, but we might need to make that a separate PR since this one is already over the line limit. Thoughts about making a follow up ticket for that work?

Reply:
Sure, we could do that.

Reply:
Meh, I went ahead and separated it out into a `TravelPay::BaseClient`.